src/views/Player.vue

```diff
@@ -19,22 +19,42 @@
         v-if="isQuestionShown"
         data-test="modal"
       ></QuestionModal>
+
+      <Scorecard
+        id="scorecardmodal"
+        class="absolute z-10"
+        :class="{
+          hidden: !isScorecardShown,
+        }"
+        :metrics="scorecardMetrics"
+        :progressPercentage="scorecardProgress"
+        :isShown="isScorecardShown"
+        :title="title"
+        greeting="Hooray! Congrats on completing the quiz! 🎉"
+        :numQuestionsAnswered="numQuestionsAnswered"
+        :hasGradedQuestions="hasGradedQuestions"
+        @go-back="goToLastQuestion"
```

Comment:
Correct me if I'm wrong, but I feel like they don't mean the same thing. 'Go back' vs 'go to the last question'. Maybe go to previous question?